pkg/artifactory/resource_artifactory_permission_targets_v1_test.go

```diff
@@ -52,6 +52,10 @@ func testPermissionTargetV1CheckDestroy(id string) func(*terraform.State) error
 
 		permissionTargets, resp, err := client.V1.Security.GetPermissionTargets(context.Background(), rs.Primary.ID)
 
+		if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.